Cesium.CodeGen/Ir/Types/PrimitiveType.cs

```diff
@@ -149,6 +149,7 @@ Kind switch
                 TargetArchitectureSet.Bit64 => 8,
                 _ => throw new AssertException($"Architecture set not supported: {arch}.")
             },
+            PrimitiveTypeKind.Void => 1,
```

Comment:
Welp, this one is weird. Technically we shouldn't define `sizeof(void)`. Why is that required?

Reply:
```
void* ptr;
ptr += 8;
```

Reply:
Oh wow, GCC also thinks `sizeof(void)` is 1. Okay.